EventEmmiter/eventemmiter.js

```diff
@@ -0,0 +1,68 @@
+class EventEmitter {
+  constructor() {
+    //maintain array with event nae and all listeners which have subscribed to the event
+    this.events = {};
+  }
+
+  subscribe(eventName, callback) {
+    //here our callback is listener
+
+    if (typeof callback !== "function") {
+      throw new Error("Listener must be a function!");
+    }
+
+    //check if event present or not if not
+    //create new event and initialize with empty array
+
+    this.events[eventName] = this.events[eventName] || [];
+    // console.log(eventName);
+
+    //now push list callback(listener) to that particular event array
+    this.events[eventName].push(callback);
+
+    return {
+      unsubscribe: () => {
+        //get the index of event from particular event array
+        const index = this.events[eventName].indexOf(callback);
+        this.events[eventName].splice(index, 1);
+        return undefined;
```

Comment:
```suggestion
        return undefined;
```
in javascript if you do not write return statement, it will return undefined by default.